website/sidebars.json

```diff
@@ -1,194 +1,173 @@
-                    "integrator-section/verifiable-document/ethereum/issuing-document",
-                    "integrator-section/verifiable-document/ethereum/revoking-document"
-                  ]
-                },
-                {
-                  "label": "DID",
-                  "type": "category",
-                  "items": [
-                    "integrator-section/verifiable-document/did/create",
-                    "integrator-section/verifiable-document/did/document-store-or-ocsp",
-                    "integrator-section/verifiable-document/did/dns",
-                    "integrator-section/verifiable-document/did/raw-document-did",
-                    "integrator-section/verifiable-document/did/wrapping-document-did",
-                    "integrator-section/verifiable-document/did/signing-document",
-                    "integrator-section/verifiable-document/did/revoking-document-did"
-                  ]
-                },
-                "integrator-section/verifiable-document/differences",
-                "integrator-section/verifiable-document/going-further"
-              ]
-            },
-            {
-              "type": "category",
-              "label": "Transferable Record",
-              "items": [
-                "integrator-section/transferable-record/overview-tr",
-                "integrator-section/transferable-record/token-registry",
-                "integrator-section/transferable-record/preparing-transferable-record",
-                "integrator-section/transferable-record/issuing-transferable-record",
-                "integrator-section/transferable-record/going-further-tr"
-              ]
-            }
-          ]
-        },
-        {
-          "type": "category",
-          "label": "Web App Tutorial",
-          "items": [
-            "integrator-section/webapp-tutorial/deploy-document-store-webapp",
-            "integrator-section/webapp-tutorial/issue-document-store-webapp",
-            "integrator-section/webapp-tutorial/revoke-document-store-webapp"
-          ]
-        }
-      ]
-    }
+
+  "revoke": [
+      "revoke-section/revoke",
+      "revoke-section/diff-btw-revocation-methods"
   ],
-  "developer": [
-    {
-      "type": "category",
-      "label": "Quickstart (Code)",
-      "items": [
-        "developer-section/quickstart/create-verifiable-document-issuer",
-        "developer-section/quickstart/create-custom-renderer",
-        "developer-section/quickstart/create-custom-schema",
-        "developer-section/quickstart/configure-dns",
-        "developer-section/quickstart/oa-embedded-html",
-        "developer-section/quickstart/oa-embedded-qrcode"
-      ]
-    },
-    {
+
+  "verify": [
+    "verify-section/verify",
+    "verify-section/main-properties",
+    "verify-section/document-integrity",
+    "verify-section/issuance-status",
+    "verify-section/issuance-identity"
```

Comment:
can we nest these 3 sections under `main-properties` and have it expanded by default?

Reply:
Yes... When the 3 articles are nested under the category called main properties, it can no longer hold any content. 
So the article with `id: main-properties` has its sidebar label and title modified to fit into the category as an overview:

> `title: About the verification properties`
> `sidebar_label: Overview`